src/jdk.crypto.cryptoki/share/native/libj2pkcs11/pkcs11t.h

```diff
@@ -2173,6 +2179,16 @@ typedef struct CK_TLS12_MASTER_KEY_DERIVE_PARAMS {
 typedef CK_TLS12_MASTER_KEY_DERIVE_PARAMS CK_PTR \
         CK_TLS12_MASTER_KEY_DERIVE_PARAMS_PTR;
 
+typedef struct CK_NSS_TLS_EXTENDED_MASTER_KEY_DERIVE_PARAMS {
+    CK_MECHANISM_TYPE   prfHashMechanism;
+    CK_BYTE_PTR         pSessionHash;
+    CK_ULONG            ulSessionHashLen;
+    CK_VERSION_PTR      pVersion;
```

Comment:
Other structures align the field names just one space after the longest member type.